mitodl/odl-video-service
# Update USwitch component

#### What are the relevant tickets?
Fixes #108
Depends on https://github.com/mitodl/omniplayer/pull/1

#### What's this PR do?
Updates front-end code to work with the latest version of USwitch, which works with any .mit.edu subdomain.

#### How should this be manually tested?
- Set `USWITCH_URL=https://s3.amazonaws.com/odl-video-service-uswitch-dev/dev`
- Change a video to `multiangle=true` in the admin console
- Temporarily modify your hosts file so that 'video.odl.mit.edu' points to your docker IP
- Load that video's detail page.  The video should play with the USwitch player.
